### PR TITLE
POC for #99

### PR DESCRIPTION
**Pull request by timorunge**

### CODE REVIEW

**Comment by timorunge** [comment]
@gammons it's breaking formatting a little bit. Do you have any idea why?

**Comment by timorunge** [comment]
Actually the problem is the tabwriter and how the tabwriter is handling color codes. I was taking a look at the [tablewriter](https://github.com/olekukonko/tablewriter) but that would be a bigger change (and some of the things are not easily possible).

**Comment by timorunge** [comment]
Additional reference: https://stackoverflow.com/questions/35398497/how-do-i-get-colors-to-work-with-golang-tabwriter

File: ultralist/screen_printer.go [hunk 1]
```diff
@@ -101,6 +102,21 @@ func (f *ScreenPrinter) formatDue(due string, isPriority bool, completed bool) s
...
+		information = append(information, " ")
+	}
+	if todo.HasNotes() {
+		information = append(information, "N")
+	} else {
+		information = append(information, " ")
+	}
+	return white.Sprint(strings.Join(information, " "))
+}
+
```
Comment (gammons) [inline ultralist/screen_printer.go]: should we also have a status for archived?

Comment (timorunge) [reply ultralist/screen_printer.go]: Yeah, sure, that's an good idea.

Comment (timorunge) [reply ultralist/screen_printer.go]: Added the A flag and re-merged with master after #180.

**Comment by gammons** [comment]
I've been messing around with this but I am not sure if I like the huge gap between the checkbox column and the date column.  But I'm not sure how else we could show these flags.

**Comment by timorunge** [comment]
> I've been messing around with this but I am not sure if I like the huge gap between the checkbox column and the date column. But I'm not sure how else we could show these flags.

Maybe working with a flag (e.g. `--displayInformation`)? I had this idea while working on #168 but wanted to keep it simple and separated for this POC.

**Comment by gammons** [comment]
opened #203 to replace this.

**Pull request closed without merging**